### ISSUE TITLE
Please revert to previous tab (container) type indicator

### ISSUE THREAD

**Issue by jumapico** [mozilla/testpilot-containers#739]
After an automatic update to 3.1.0 version, i see a bad change (personal feeling) in the container indicator.

The new indicator has less visibility; i need more effort to see it:

![new-worst-tab-indicator](https://user-images.githubusercontent.com/149845/29417868-0472cda4-8341-11e7-8456-6cea8f4bc516.png)

The advertised indicator in the test pilot page is better for visibility; i prefer it due it need less effort to identify the tab (container) type:

![advertised-tab-indicator](https://user-images.githubusercontent.com/149845/29417895-19c20d6e-8341-11e7-8fb3-4856ecf7f64f.png)
---

about:support:

```
{
  "application": {
    "name": "Firefox",
    "osVersion": "Linux 4.11.0-1-amd64",
    "version": "54.0.1",
    "buildID": "20170628145605",
    "userAgent": "Mozilla/5.0 (X11; Linux x86_64; rv:54.0) Gecko/20100101 Firefox/54.0",
    "safeMode": false,
    "updateChannel": "release",
    "supportURL": "https://support.mozilla.org/1/firefox/54.0.1/Linux/en-US/",
    "numTotalWindows": 1,
    "numRemoteWindows": 1,
    "remoteAutoStart": true,
    "autoStartStatus": 1,
    "keyGoogleFound": true,
    "keyMozillaFound": true
  },
  "modifiedPreferences": {
    "accessibility.typeaheadfind.flashBar": 0,
    "accessibility.blockautorefresh": true,
    "browser.cache.disk.smart_size.first_run": false,
    "browser.cache.disk.smart_size.use_old_max": false,
    "browser.cache.disk.hashstats_reported": 1,
    "browser.cache.disk.filesystem_reported": 1,
    "browser.cache.disk.capacity": 358400,
    "browser.cache.frecency_experiment": 4,
    "browser.cache.memory.max_entry_size": 10240,
    "browser.display.use_document_fonts": 0,
    "browser.download.importedFromSqlite": true,
    "browser.download.useDownloadDir": false,
    "browser.fixup.domainwhitelist.10.16.0.67": true,
    "browser.places.smartBookmarksVersion": 8,
    "browser.sessionstore.upgradeBackup.latestBuildID": "20170628145605",
    "browser.startup.homepage_override.mstone": "54.0.1",
    "browser.startup.homepage": "about:home",
    "browser.startup.homepage_override.buildID": "20170628145605",
    "browser.tabs.warnOnOpen": false,
    "browser.tabs.remote.autostart.2": true,
    "browser.tabs.animate": false,
    "browser.urlbar.suggest.searches": true,
    "browser.urlbar.searchSuggestionsChoice": true,
    "browser.urlbar.userMadeSearchSuggestionsChoice": true,
    "dom.push.userAgentID": "bfe1ddbf72a04505bd98964f8a910ece",
    "dom.apps.reset-permissions": true,
    "dom.mozApps.used": true,
    "extensions.lastAppVersion": "54.0.1",
    "font.name.serif.x-western": "Courier 10 Pitch",
    "font.name.sans-serif.x-western": "Courier 10 Pitch",
    "font.name.monospace.x-western": "Courier 10 Pitch",
    "font.internaluseonly.changed": true,
    "gfx.crash-guard.glcontext.driverVersion": "3.0 Mesa 13.0.2",
    "gfx.crash-guard.glcontext.deviceID": "Mesa DRI Intel(R) Ivybridge Desktop ",
    "gfx.crash-guard.glcontext.appVersion": "45.6.0",
    "gfx.crash-guard.status.glcontext": 2,
    "media.eme.enabled": true,
    "media.webrtc.debug.aec_log_dir": "/tmp",
    "media.webrtc.debug.log_file": "/tmp/WebRTC.log",
    "media.gmp-widevinecdm.lastUpdate": 1490211860,
    "media.gmp-gmpopenh264.lastUpdate": 1487783522,
    "media.peerconnection.ice.default_address_only": true,
    "media.gmp-gmpopenh264.version": "1.6",
    "media.gmp-widevinecdm.abi": "x86_64-gcc3",
    "media.gmp-manager.lastCheck": 1502891388,
    "media.gmp-widevinecdm.version": "1.4.8.903",
    "media.youtube-ua.override.to": "43",
    "media.gmp.storage.version.observed": 1,
    "media.gmp-gmpopenh264.abi": "x86_64-gcc3",
    "media.peerconnection.ice.no_host": true,
    "media.gmp-manager.buildID": "20170628145605",
    "network.predictor.cleaned-up": true,
    "network.cookie.prefsMigrated": true,
    "places.history.expiration.transient_current_max_pages": 126771,
    "places.database.lastMaintenance": 1502891494,
    "plugin.importedState": true,
    "plugin.disable_full_page_plugin_for_types": "application/pdf",
    "print.print_margin_top": "0.5",
    "print.print_paper_width": "  8.50",
    "print.print_paper_size_unit": 0,
    "print.print_to_file": false,
    "print.print_orientation": 0,
    "print.print_paper_height": " 11.00",
    "print.print_scaling": "  1.00",
    "print.print_unwriteable_margin_right": 25,
    "print.print_unwriteable_margin_left": 25,
    "print.print_paper_data": 0,
    "print.print_resolution": 300,
    "print.print_unwriteable_margin_bottom": 56,
    "print.print_paper_name": "na_letter",
    "print.print_oddpages": true,
    "print.print_bgimages": false,
    "print.print_duplex": 0,
    "print.print_evenpages": true,
    "print.print_margin_left": "0.5",
    "print.print_paper_size_type": 1,
    "print.print_shrink_to_fit": true,
    "print.print_margin_right": "0.5",
    "print.print_unwriteable_margin_top": 25,
    "print.print_bgcolor": false,
    "print.print_margin_bottom": "0.5",
    "print.print_in_color": true,
    "print.print_page_delay": 50,
    "privacy.cpd.siteSettings": true,
    "privacy.userContext.ui.enabled": true,
    "privacy.sanitize.timeSpan": 0,
    "privacy.sanitize.migrateClearSavedPwdsOnExit": true,
    "privacy.cpd.sessions": false,
    "privacy.cpd.cookies": false,
    "privacy.usercontext.about_newtab_segregation.enabled": true,
    "privacy.userContext.enabled": true,
    "privacy.donottrackheader.enabled": true,
    "privacy.userContext.longPressBehavior": 2,
    "privacy.cpd.offlineApps": true,
    "privacy.sanitize.migrateFx3Prefs": true,
    "security.webauth.softtoken_counter": 1,
    "security.disable_button.openDeviceManager": false,
    "security.disable_button.openCertManager": false,
    "security.webauth.u2f": true,
    "security.webauth.u2f_enable_softtoken": true,
    "security.webauth.u2f_enable_usbtoken": true,
    "services.sync.declinedEngines": "passwords",
    "services.sync.lastPing": 1502978929,
    "services.sync.lastSync": "Thu Aug 17 2017 11:39:10 GMT-0300",
    "services.sync.numClients": 2,
    "services.sync.engine.greasemonkey": true,
    "services.sync.engine.passwords": false,
    "services.sync.engine.prefs.modified": false,
    "storage.vacuum.last.index": 1,
    "storage.vacuum.last.places.sqlite": 1501865580,
    "webgl.force-enabled": true
  },
  "lockedPreferences": {},
  "javaScript": {
    "incrementalGCEnabled": true
  },
  "accessibility": {
    "isActive": false,
    "forceDisabled": 0
  },
  "libraryVersions": {
    "NSPR": {
      "minVersion": "4.14",
      "version": "4.14"
    },
    "NSS": {
      "minVersion": "3.30.2",
      "version": "3.30.2"
    },
    "NSSUTIL": {
      "minVersion": "3.30.2",
      "version": "3.30.2"
    },
    "NSSSSL": {
      "minVersion": "3.30.2",
      "version": "3.30.2"
    },
    "NSSSMIME": {
      "minVersion": "3.30.2",
      "version": "3.30.2"
    }
  },
  "userJS": {
    "exists": false
  },
  "crashes": {
    "submitted": [],
    "pending": 0
  },
  "sandbox": {
    "hasSeccompBPF": true,
    "hasSeccompTSync": true,
    "hasPrivilegedUserNamespaces": true,
    "hasUserNamespaces": false,
    "canSandboxContent": true,
    "canSandboxMedia": true,
    "syscallLog": [],
    "contentSandboxLevel": 2
  },
  "experiments": [],
  "extensions": [
    {
      "name": "Application Update Service Helper",
      "version": "2.0",
      "isActive": true,
      "id": "aushelper@mozilla.org"
    },
    {
      "name": "Firefox Notes",
      "version": "1.6.3dev",
      "isActive": true,
      "id": "notes@mozilla.com"
    },
    {
      "name": "Firefox Screenshots",
      "version": "6.6.0",
      "isActive": true,
      "id": "screenshots@mozilla.org"
    },
    {
      "name": "Follow-on Search Telemetry",
      "version": "0.9.1",
      "isActive": true,
      "id": "followonsearch@mozilla.com"
    },
    {
      "name": "GNOME Shell integration",
      "version": "8.2.1",
      "isActive": true,
      "id": "chrome-gnome-shell@gnome.org"
    },
    {
      "name": "Greasemonkey",
      "version": "3.11",
      "isActive": true,
      "id": "{e4a8a97b-f2ed-450b-b12d-ee082ba24781}"
    },
    {
      "name": "JSON Lite",
      "version": "0.20",
      "isActive": true,
      "id": "{16a49f65-1369-4839-a5ef-db2581e08b16}"
    },
    {
      "name": "Mailvelope",
      "version": "1.8.1",
      "isActive": true,
      "id": "jid1-AQqSMBYb0a8ADg@jetpack"
    },
    {
      "name": "Multi-process staged rollout",
      "version": "1.85",
      "isActive": true,
      "id": "e10srollout@mozilla.org"
    },
    {
      "name": "Pocket",
      "version": "1.0.5",
      "isActive": true,
      "id": "firefox@getpocket.com"
    },
    {
      "name": "Privacy Badger",
      "version": "2017.7.24",
      "isActive": true,
      "id": "jid1-MnnxcxisBPnSXQ@jetpack"
    },
    {
      "name": "Shield Recipe Client",
      "version": "1.0.0",
      "isActive": true,
      "id": "shield-recipe-client@mozilla.org"
    },
    {
      "name": "Tab Center",
      "version": "1.36.0",
      "isActive": true,
      "id": "tabcentertest1@mozilla.com"
    },
    {
      "name": "Test Pilot",
      "version": "2.0.2-dev-0d66c35",
      "isActive": true,
      "id": "@testpilot-addon"
    },
    {
      "name": "Testpilot containers",
      "version": "3.1.0",
      "isActive": true,
      "id": "@testpilot-containers"
    },
    {
      "name": "VimFx",
      "version": "0.20.10",
      "isActive": true,
      "id": "VimFx@akhodakivskiy.github.com"
    },
    {
      "name": "Web Compat",
      "version": "1.1",
      "isActive": true,
      "id": "webcompat@mozilla.org"
    },
    {
      "name": "Bookmark All",
      "version": "1.3.2.1-signed.1-signed",
      "isActive": false,
      "id": "{a76cd07b-f0d7-4ef9-9566-8faef6e290e4}"
    },
    {
      "name": "cors everywhere",
      "version": "20150910-1410",
      "isActive": false,
      "id": "cors-everywhere@spenibus"
    },
    {
      "name": "Firebug",
      "version": "2.0.19",
      "isActive": false,
      "id": "firebug@software.joehewitt.com"
    },
    {
      "name": "Markdown Viewer",
      "version": "1.12",
      "isActive": false,
      "id": "markdownviewer@thiht.fr"
    }
  ],
  "graphics": {
    "numTotalWindows": 1,
    "numAcceleratedWindows": 0,
    "windowLayerManagerType": "Basic",
    "windowLayerManagerRemote": true,
    "currentAudioBackend": "pulse",
    "numAcceleratedWindowsMessage": [
      ""
    ],
    "adapterDescription": "Intel Open Source Technology Center -- Mesa DRI Intel(R) Ivybridge Desktop ",
    "adapterVendorID": "Intel Open Source Technology Center",
    "adapterDeviceID": "Mesa DRI Intel(R) Ivybridge Desktop ",
    "adapterRAM": "",
    "adapterDrivers": "",
    "driverVersion": "3.0 Mesa 13.0.6",
    "driverDate": "",
    "webgl1Renderer": "Intel Open Source Technology Center -- Mesa DRI Intel(R) Ivybridge Desktop ",
    "webgl1Version": "3.0 Mesa 13.0.6",
    "webgl1DriverExtensions": "GL_ARB_multisample GL_EXT_abgr GL_EXT_bgra GL_EXT_blend_color GL_EXT_blend_minmax GL_EXT_blend_subtract GL_EXT_copy_texture GL_EXT_polygon_offset GL_EXT_subtexture GL_EXT_texture_object GL_EXT_vertex_array GL_EXT_compiled_vertex_array GL_EXT_texture GL_EXT_texture3D GL_IBM_rasterpos_clip GL_ARB_point_parameters GL_EXT_draw_range_elements GL_EXT_packed_pixels GL_EXT_point_parameters GL_EXT_rescale_normal GL_EXT_separate_specular_color GL_EXT_texture_edge_clamp GL_SGIS_generate_mipmap GL_SGIS_texture_border_clamp GL_SGIS_texture_edge_clamp GL_SGIS_texture_lod GL_ARB_framebuffer_sRGB GL_ARB_multitexture GL_EXT_framebuffer_sRGB GL_IBM_multimode_draw_arrays GL_IBM_texture_mirrored_repeat GL_3DFX_texture_compression_FXT1 GL_ARB_texture_cube_map GL_ARB_texture_env_add GL_ARB_transpose_matrix GL_EXT_blend_func_separate GL_EXT_fog_coord GL_EXT_multi_draw_arrays GL_EXT_secondary_color GL_EXT_texture_env_add GL_EXT_texture_filter_anisotropic GL_EXT_texture_lod_bias GL_INGR_blend_func_separate GL_NV_blend_square GL_NV_light_max_exponent GL_NV_texgen_reflection GL_NV_texture_env_combine4 GL_S3_s3tc GL_SUN_multi_draw_arrays GL_ARB_texture_border_clamp GL_ARB_texture_compression GL_EXT_framebuffer_object GL_EXT_texture_compression_s3tc GL_EXT_texture_env_combine GL_EXT_texture_env_dot3 GL_MESA_window_pos GL_NV_packed_depth_stencil GL_NV_texture_rectangle GL_ARB_depth_texture GL_ARB_occlusion_query GL_ARB_shadow GL_ARB_texture_env_combine GL_ARB_texture_env_crossbar GL_ARB_texture_env_dot3 GL_ARB_texture_mirrored_repeat GL_ARB_window_pos GL_EXT_stencil_two_side GL_EXT_texture_cube_map GL_NV_depth_clamp GL_APPLE_packed_pixels GL_APPLE_vertex_array_object GL_ARB_draw_buffers GL_ARB_fragment_program GL_ARB_fragment_shader GL_ARB_shader_objects GL_ARB_vertex_program GL_ARB_vertex_shader GL_ATI_draw_buffers GL_ATI_texture_env_combine3 GL_ATI_texture_float GL_EXT_shadow_funcs GL_EXT_stencil_wrap GL_MESA_pack_invert GL_NV_primitive_restart GL_ARB_depth_clamp GL_ARB_fragment_program_shadow GL_ARB_half_float_pixel GL_ARB_occlusion_query2 GL_ARB_point_sprite GL_ARB_shading_language_100 GL_ARB_sync GL_ARB_texture_non_power_of_two GL_ARB_vertex_buffer_object GL_ATI_blend_equation_separate GL_EXT_blend_equation_separate GL_OES_read_format GL_ARB_color_buffer_float GL_ARB_pixel_buffer_object GL_ARB_texture_compression_rgtc GL_ARB_texture_float GL_ARB_texture_rectangle GL_EXT_packed_float GL_EXT_pixel_buffer_object GL_EXT_texture_compression_dxt1 GL_EXT_texture_compression_rgtc GL_EXT_texture_rectangle GL_EXT_texture_sRGB GL_EXT_texture_shared_exponent GL_ARB_framebuffer_object GL_EXT_framebuffer_blit GL_EXT_framebuffer_multisample GL_EXT_packed_depth_stencil GL_APPLE_object_purgeable GL_ARB_vertex_array_object GL_ATI_separate_stencil GL_EXT_draw_buffers2 GL_EXT_draw_instanced GL_EXT_gpu_program_parameters GL_EXT_texture_array GL_EXT_texture_integer GL_EXT_texture_sRGB_decode GL_EXT_timer_query GL_OES_EGL_image GL_AMD_performance_monitor GL_ARB_copy_buffer GL_ARB_depth_buffer_float GL_ARB_draw_instanced GL_ARB_half_float_vertex GL_ARB_instanced_arrays GL_ARB_map_buffer_range GL_ARB_texture_rg GL_ARB_texture_swizzle GL_ARB_vertex_array_bgra GL_EXT_texture_swizzle GL_EXT_vertex_array_bgra GL_NV_conditional_render GL_AMD_conservative_depth GL_AMD_draw_buffers_blend GL_AMD_seamless_cubemap_per_texture GL_ARB_ES2_compatibility GL_ARB_blend_func_extended GL_ARB_debug_output GL_ARB_draw_buffers_blend GL_ARB_draw_elements_base_vertex GL_ARB_explicit_attrib_location GL_ARB_fragment_coord_conventions GL_ARB_provoking_vertex GL_ARB_sample_shading GL_ARB_sampler_objects GL_ARB_seamless_cube_map GL_ARB_shader_texture_lod GL_ARB_texture_cube_map_array GL_ARB_texture_gather GL_ARB_texture_multisample GL_ARB_texture_query_lod GL_ARB_texture_rgb10_a2ui GL_ARB_uniform_buffer_object GL_ARB_vertex_type_2_10_10_10_rev GL_EXT_provoking_vertex GL_EXT_texture_snorm GL_MESA_texture_signed_rgba GL_NV_texture_barrier GL_ARB_get_program_binary GL_ARB_robustness GL_ARB_separate_shader_objects GL_ARB_shader_bit_encoding GL_ARB_texture_compression_bptc GL_ARB_timer_query GL_ARB_transform_feedback2 GL_ARB_transform_feedback3 GL_ANGLE_texture_compression_dxt3 GL_ANGLE_texture_compression_dxt5 GL_ARB_compressed_texture_pixel_storage GL_ARB_conservative_depth GL_ARB_internalformat_query GL_ARB_map_buffer_alignment GL_ARB_shader_atomic_counters GL_ARB_shader_image_load_store GL_ARB_shading_language_420pack GL_ARB_shading_language_packing GL_ARB_texture_storage GL_ARB_transform_feedback_instanced GL_EXT_framebuffer_multisample_blit_scaled GL_EXT_transform_feedback GL_AMD_shader_trinary_minmax GL_ARB_ES3_compatibility GL_ARB_arrays_of_arrays GL_ARB_clear_buffer_object GL_ARB_compute_shader GL_ARB_copy_image GL_ARB_explicit_uniform_location GL_ARB_framebuffer_no_attachments GL_ARB_invalidate_subdata GL_ARB_program_interface_query GL_ARB_shader_image_size GL_ARB_shader_storage_buffer_object GL_ARB_texture_query_levels GL_ARB_texture_storage_multisample GL_ARB_texture_view GL_ARB_vertex_attrib_binding GL_KHR_debug GL_KHR_robustness GL_ARB_buffer_storage GL_ARB_clear_texture GL_ARB_internalformat_query2 GL_ARB_multi_bind GL_ARB_seamless_cubemap_per_texture GL_ARB_shader_draw_parameters GL_ARB_texture_mirror_clamp_to_edge GL_ARB_vertex_type_10f_11f_11f_rev GL_EXT_shader_integer_mix GL_INTEL_performance_query GL_ARB_clip_control GL_ARB_conditional_render_inverted GL_ARB_cull_distance GL_ARB_derivative_control GL_ARB_get_texture_sub_image GL_ARB_pipeline_statistics_query GL_ARB_shader_texture_image_samples GL_ARB_texture_barrier GL_EXT_polygon_offset_clamp GL_KHR_blend_equation_advanced GL_KHR_context_flush_control GL_ARB_shader_atomic_counter_ops GL_ARB_shader_clock GL_EXT_shader_samples_identical GL_MESA_shader_integer_functions ",
    "webgl1Extensions": "ANGLE_instanced_arrays EXT_blend_minmax EXT_color_buffer_half_float EXT_frag_depth EXT_sRGB EXT_shader_texture_lod EXT_texture_filter_anisotropic EXT_disjoint_timer_query MOZ_debug_get OES_element_index_uint OES_standard_derivatives OES_texture_float OES_texture_float_linear OES_texture_half_float OES_texture_half_float_linear OES_vertex_array_object WEBGL_color_buffer_float WEBGL_compressed_texture_etc WEBGL_compressed_texture_s3tc WEBGL_debug_renderer_info WEBGL_debug_shaders WEBGL_depth_texture WEBGL_draw_buffers WEBGL_lose_context MOZ_WEBGL_lose_context MOZ_WEBGL_compressed_texture_s3tc MOZ_WEBGL_depth_texture",
    "webgl1WSIInfo": "GLX 1.4\nGLX_VENDOR(client): Mesa Project and SGI\nGLX_VENDOR(server): SGI\nExtensions: GLX_ARB_create_context GLX_ARB_create_context_profile GLX_ARB_create_context_robustness GLX_ARB_fbconfig_float GLX_ARB_framebuffer_sRGB GLX_ARB_get_proc_address GLX_ARB_multisample GLX_EXT_import_context GLX_EXT_visual_info GLX_EXT_visual_rating GLX_EXT_fbconfig_packed_float GLX_EXT_framebuffer_sRGB GLX_EXT_create_context_es2_profile GLX_EXT_create_context_es_profile GLX_MESA_copy_sub_buffer GLX_MESA_multithread_makecurrent GLX_MESA_query_renderer GLX_MESA_swap_control GLX_OML_swap_method GLX_OML_sync_control GLX_SGI_make_current_read GLX_SGI_swap_control GLX_SGI_video_sync GLX_SGIS_multisample GLX_SGIX_fbconfig GLX_SGIX_pbuffer GLX_SGIX_visual_select_group GLX_EXT_texture_from_pixmap GLX_INTEL_swap_event GLX_EXT_buffer_age ",
    "webgl2Renderer": "Intel Open Source Technology Center -- Mesa DRI Intel(R) Ivybridge Desktop ",
    "webgl2Version": "3.3 (Core Profile) Mesa 13.0.6",
    "webgl2DriverExtensions": "GL_3DFX_texture_compression_FXT1 GL_AMD_conservative_depth GL_AMD_draw_buffers_blend GL_AMD_performance_monitor GL_AMD_seamless_cubemap_per_texture GL_AMD_shader_trinary_minmax GL_AMD_vertex_shader_layer GL_AMD_vertex_shader_viewport_index GL_ANGLE_texture_compression_dxt3 GL_ANGLE_texture_compression_dxt5 GL_APPLE_object_purgeable GL_ARB_ES2_compatibility GL_ARB_ES3_compatibility GL_ARB_arrays_of_arrays GL_ARB_base_instance GL_ARB_blend_func_extended GL_ARB_buffer_storage GL_ARB_clear_buffer_object GL_ARB_clear_texture GL_ARB_clip_control GL_ARB_compressed_texture_pixel_storage GL_ARB_compute_shader GL_ARB_conditional_render_inverted GL_ARB_conservative_depth GL_ARB_copy_buffer GL_ARB_copy_image GL_ARB_cull_distance GL_ARB_debug_output GL_ARB_depth_buffer_float GL_ARB_depth_clamp GL_ARB_derivative_control GL_ARB_direct_state_access GL_ARB_draw_buffers GL_ARB_draw_buffers_blend GL_ARB_draw_elements_base_vertex GL_ARB_draw_indirect GL_ARB_draw_instanced GL_ARB_enhanced_layouts GL_ARB_explicit_attrib_location GL_ARB_explicit_uniform_location GL_ARB_fragment_coord_conventions GL_ARB_fragment_layer_viewport GL_ARB_fragment_shader GL_ARB_framebuffer_no_attachments GL_ARB_framebuffer_object GL_ARB_framebuffer_sRGB GL_ARB_get_program_binary GL_ARB_get_texture_sub_image GL_ARB_gpu_shader5 GL_ARB_half_float_pixel GL_ARB_half_float_vertex GL_ARB_instanced_arrays GL_ARB_internalformat_query GL_ARB_internalformat_query2 GL_ARB_invalidate_subdata GL_ARB_map_buffer_alignment GL_ARB_map_buffer_range GL_ARB_multi_bind GL_ARB_multi_draw_indirect GL_ARB_occlusion_query2 GL_ARB_pipeline_statistics_query GL_ARB_pixel_buffer_object GL_ARB_point_sprite GL_ARB_program_interface_query GL_ARB_provoking_vertex GL_ARB_robustness GL_ARB_sample_shading GL_ARB_sampler_objects GL_ARB_seamless_cube_map GL_ARB_seamless_cubemap_per_texture GL_ARB_separate_shader_objects GL_ARB_shader_atomic_counter_ops GL_ARB_shader_atomic_counters GL_ARB_shader_bit_encoding GL_ARB_shader_clock GL_ARB_shader_draw_parameters GL_ARB_shader_image_load_store GL_ARB_shader_image_size GL_ARB_shader_objects GL_ARB_shader_storage_buffer_object GL_ARB_shader_subroutine GL_ARB_shader_texture_image_samples GL_ARB_shader_texture_lod GL_ARB_shader_viewport_layer_array GL_ARB_shading_language_420pack GL_ARB_shading_language_packing GL_ARB_sync GL_ARB_tessellation_shader GL_ARB_texture_barrier GL_ARB_texture_buffer_object GL_ARB_texture_buffer_object_rgb32 GL_ARB_texture_buffer_range GL_ARB_texture_compression_bptc GL_ARB_texture_compression_rgtc GL_ARB_texture_cube_map_array GL_ARB_texture_float GL_ARB_texture_gather GL_ARB_texture_mirror_clamp_to_edge GL_ARB_texture_multisample GL_ARB_texture_non_power_of_two GL_ARB_texture_query_levels GL_ARB_texture_query_lod GL_ARB_texture_rectangle GL_ARB_texture_rg GL_ARB_texture_rgb10_a2ui GL_ARB_texture_storage GL_ARB_texture_storage_multisample GL_ARB_texture_swizzle GL_ARB_texture_view GL_ARB_timer_query GL_ARB_transform_feedback2 GL_ARB_transform_feedback3 GL_ARB_transform_feedback_instanced GL_ARB_uniform_buffer_object GL_ARB_vertex_array_bgra GL_ARB_vertex_array_object GL_ARB_vertex_attrib_binding GL_ARB_vertex_shader GL_ARB_vertex_type_10f_11f_11f_rev GL_ARB_vertex_type_2_10_10_10_rev GL_ARB_viewport_array GL_ATI_blend_equation_separate GL_ATI_texture_float GL_EXT_abgr GL_EXT_blend_equation_separate GL_EXT_draw_buffers2 GL_EXT_draw_instanced GL_EXT_framebuffer_blit GL_EXT_framebuffer_multisample GL_EXT_framebuffer_multisample_blit_scaled GL_EXT_framebuffer_sRGB GL_EXT_packed_depth_stencil GL_EXT_packed_float GL_EXT_pixel_buffer_object GL_EXT_polygon_offset_clamp GL_EXT_provoking_vertex GL_EXT_shader_integer_mix GL_EXT_shader_samples_identical GL_EXT_texture_array GL_EXT_texture_compression_dxt1 GL_EXT_texture_compression_rgtc GL_EXT_texture_compression_s3tc GL_EXT_texture_filter_anisotropic GL_EXT_texture_integer GL_EXT_texture_sRGB GL_EXT_texture_sRGB_decode GL_EXT_texture_shared_exponent GL_EXT_texture_snorm GL_EXT_texture_swizzle GL_EXT_timer_query GL_EXT_transform_feedback GL_EXT_vertex_array_bgra GL_IBM_multimode_draw_arrays GL_INTEL_performance_query GL_KHR_blend_equation_advanced GL_KHR_context_flush_control GL_KHR_debug GL_KHR_robustness GL_MESA_pack_invert GL_MESA_shader_integer_functions GL_MESA_texture_signed_rgba GL_NV_conditional_render GL_NV_depth_clamp GL_NV_packed_depth_stencil GL_NV_texture_barrier GL_OES_EGL_image GL_S3_s3tc",
    "webgl2Extensions": "EXT_color_buffer_float EXT_texture_filter_anisotropic EXT_disjoint_timer_query MOZ_debug_get OES_texture_float_linear WEBGL_compressed_texture_etc WEBGL_compressed_texture_s3tc WEBGL_debug_renderer_info WEBGL_debug_shaders WEBGL_lose_context MOZ_WEBGL_lose_context MOZ_WEBGL_compressed_texture_s3tc",
    "webgl2WSIInfo": "GLX 1.4\nGLX_VENDOR(client): Mesa Project and SGI\nGLX_VENDOR(server): SGI\nExtensions: GLX_ARB_create_context GLX_ARB_create_context_profile GLX_ARB_create_context_robustness GLX_ARB_fbconfig_float GLX_ARB_framebuffer_sRGB GLX_ARB_get_proc_address GLX_ARB_multisample GLX_EXT_import_context GLX_EXT_visual_info GLX_EXT_visual_rating GLX_EXT_fbconfig_packed_float GLX_EXT_framebuffer_sRGB GLX_EXT_create_context_es2_profile GLX_EXT_create_context_es_profile GLX_MESA_copy_sub_buffer GLX_MESA_multithread_makecurrent GLX_MESA_query_renderer GLX_MESA_swap_control GLX_OML_swap_method GLX_OML_sync_control GLX_SGI_make_current_read GLX_SGI_swap_control GLX_SGI_video_sync GLX_SGIS_multisample GLX_SGIX_fbconfig GLX_SGIX_pbuffer GLX_SGIX_visual_select_group GLX_EXT_texture_from_pixmap GLX_INTEL_swap_event GLX_EXT_buffer_age ",
    "info": {
      "AzureCanvasBackend": "skia",
      "AzureFallbackCanvasBackend": "none",
      "AzureContentBackend": "skia",
      "AzureCanvasAccelerated": 0,
      "CairoUseXRender": 0,
      "ApzWheelInput": 1
    },
    "featureLog": {
      "features": [
        {
          "name": "HW_COMPOSITING",
          "description": "Compositing",
          "status": "blocked",
          "log": [
            {
              "type": "default",
              "status": "blocked",
              "message": "Acceleration blocked by platform"
            }
          ]
        },
        {
          "name": "OPENGL_COMPOSITING",
          "description": "OpenGL Compositing",
          "status": "unavailable",
          "log": [
            {
              "type": "default",
              "status": "unavailable",
              "message": "Hardware compositing is disabled"
            }
          ]
        },
        {
          "name": "WEBRENDER",
          "description": "WebRender",
          "status": "unavailable",
          "log": [
            {
              "type": "default",
              "status": "available"
            },
            {
              "type": "runtime",
              "status": "unavailable",
              "message": "Build doesn't include WebRender"
            }
          ]
        }
      ],
      "fallbacks": []
    },
    "crashGuards": []
  }
}
```

**Comment by groovecoder** [comment]
Which Firefox version are you using? @jonathanKingston actually ported the better version to mozilla-central so it will be the permanent implementation starting in, I believe, Firefox 57.

**Comment by jumapico** [comment]
Indeed, i'm using ff 54.

After install ff 57 (nightly), the indicator is the same as the advertised in the test pilot page.

![indicator-ff-57](https://user-images.githubusercontent.com/149845/29421377-45fe967c-834b-11e7-8ad3-707a0bb00c0b.png)

**NOTE: I think that 3px more is better for visibility**

**Comment by jonathanKingston** [comment]
54/55 is going to be improved to look the same; essentially this is interim styles as mentioned by @groovecoder will be fixed.

We may increase the size of the 57 version but currently that isn't a priority however it is a little inconsistent with pixel densities and also the selected tab too.

**Comment by jumapico** [comment]
Fixed in 4.0.1
